llama_index/llm_predictor/base.py

```diff
@@ -82,7 +83,7 @@ class Config:
 
     def __init__(
         self,
-        llm: Optional[LLMType] = None,
```

Comment:
curious: why this change?

Reply:
There was an issue with other parts of the code initializing LLMPredictor without passing in an LLM

If the `llm` is none, `resolve_llm` will return a mock llm 😅